docs/developer-guides/migration-guide-1.4.0.md

```diff
@@ -0,0 +1,169 @@
+fully backwards-compatible and is a preparatory step towards 1.4.0
+
+### Release new version using PC-SDK 1.4.0
+
+This version of the node is backwards-compatible with the runtime in versions 1.3.x. and is needed to support
+the runtime in version 1.4.0.
+
+1. Update all Partner Chians SDK dependencies in your `Cargo.toml` file to `1.4.0`
+2. Modify your runtime and node code to be compatible with PC-SDK version 1.4.0. Consult the release notes for 1.4.0
+for required changes.
+3. Release a new version of your node. This step depends on your release process.
+4. Upgrade nodes running the chain to the newly released version. This step depends on your deployment process.
+
+After this step, the network should be composed of nodes 1.4.0 running the runtime 1.3.1, and be ready for the matching
+runtime upgrade to 1.4.0.
+
+### New main chain state setup
+
+Because of the changes to the smart contracts and their on-chain data, a completely new set of main chain data needs
+to be set up on Cardano.
+
+1. Pick initial governance authority. This can be a newly generated or already existing main chain key set controlled
+by the Partner Chain authority.
+2. Select one of the governance authority's UTXOs to be used as the _genesis UTXO_.
+If an Ogmios instance is available, the UTXOs can be queried via RPC:
+```
+curl -H -s 'content-type: application/json' <ogmios-host>:<ogmios-port> -d '{"jsonrpc": "2.0", "method":"queryLedgerState/utxo", "params":{"addresses":["<governance authority address>"]}}' | jq
+```
+Save the value of this UTXO. It will be burned in the next step and will be used to identify the new Partner Chain
+in all future smart contracts operations.
+3. Set up initial governance by running `init-governance` command using the smart contracts version 7.0.1:
+```
+./pc-contracts-cli init-governance --network testnet \
+-k payment.skey \
+-c <genesis utxo> \
+-g <governance authority public key>
+```
+This will create a transaction burning the genesis UTXO and establish the governance authority for the Partner Chain
+identified by this UTXO.
+4. Set the D-parameter by running `insert-d-parameter`:
+```
+./pc-contracts-cli insert-d-parameter \
+-k payment.skey \
+--network testnet \
+--genesis-utxo <genesis utxo> \
+--d-parameter-permissioned-candidates-count <P value> \
+--d-parameter-registered-candidates-count <R value>
+```
+5. Set the permissioned candidates:
+```
+./pc-contracts-cli update-permissioned-candidates \
+-k payment.skey
+--network testnet \
+--genesis-utxo <genesis utxo> \
+--add-candidate <main chain pubkey>:<aura pubkey>:<grandpa pubkey> \
+...
+```
+repeating the `--add-candidate` argument for each permissioned candidate.
+
+### Re-register candidates
+
+At this point all SPOs willing to continue participating in the chain must re-register themselves with the newly created
+Partner Chain on Cardano. To do this, each SPO needs to do the following:
```

Comment:
I can see that docs are not specific about wizards interactions, so we are mostly good, because wizards do/should explain what is it asking for

Reply:
Yeah, I described it with the low-level commands because the wizard is not fully updated and we used them instead. Let's see how it looks like with the wizard then.

Reply:
I would like to see the flow with pc-cli wizards with the init-gov.  Not sure how much work is left there.